meta-ros-common/recipes-devtools/ogre/ogre_1.12.8.bb

```diff
@@ -18,7 +18,7 @@ inherit cmake features_check
 
 REQUIRED_DISTRO_FEATURES = "x11"
 
-DEPENDS = "zlib libx11 pugixml freetype virtual/libgl libglu"
+DEPENDS = "zlib libx11 pugixml freetype ${@bb.utils.contains('DISTRO_FEATURES', 'opengl', 'virtual/libgl libglu', 'opengl', d)}"
```

Comment:
If `DISTRO_FEATURES` doesn't contain `opengl`, then why add `opengl` to `DEPENDS`?

Reply:
that's indeed a mistake from copy&paste